src/openforms/js/components/admin/form_design/FormMetaFields.js

```diff
@@ -114,6 +117,32 @@ const FormMetaFields = ({ form, literals, onChange, errors={}, availableAuthPlug
                     <TextInput value={confirmText.value} onChange={onChange} maxLength="50" />
                 </Field>
             </FormRow>
+            <FormRow>
+                <Field
+                    name="form.registrationBackend"
+                    label="Select registration backend"
+                >
+                    <Select
+                        name="Registration backend"
+                        choices={availableRegistrationBackends.data}
+                        value={registrationBackend}
+                        onChange={onChange}
+                        allowBlank={true}
+                    />
+                </Field>
+            </FormRow>
+            <FormRow>
+                <Field
+                    name="form.registrationBackendOptions"
+                    label="Registration Backend Options"
+                >
+                    <TextInput value={typeof registrationBackendOptions === "string" ?
+                                        registrationBackendOptions :
+                                        JSON.stringify(registrationBackendOptions)}
+                               onChange={onChange}
+                               maxLength="1000" />
```

Comment:
This probably isn't the best implementation but it works and this will be changing in https://github.com/open-formulieren/open-forms/issues/547 so I didn't think it was worth spending time on this to make it better since this will be changed in the next PR.